cmd/dcrdata/views/windows.tmpl

```diff
@@ -19,7 +19,7 @@
             <div class="d-flex justify-content-between align-items-end">
               <span class="h2 d-flex pt-4 pb-1 pe-2">
                 Windows
-                <span class="dcricon-info fs24 ms-3 mt-2"></span>
+                <span class="dcricon-info fs24 ms-3 mt-2" title="List of Decred Windows"></span>
```

Comment:
What's a "Decred Window"? (natural user question)  There are many windows in Decred, so this isn't informative enough.
This is the stake difficulty window aka ticket price window.  The URL path is `/ticketpricewindows`.